src/backend/gpopt/translate/CTranslatorUtils.cpp

```diff
@@ -387,8 +387,8 @@ CTranslatorUtils::ResolvePolymorphicTypes(CMemoryPool *mp,
 	const ULONG num_args = std::min(num_arg_types, num_args_from_query);
 	const ULONG total_args = num_args + num_return_args;
 
-	OID arg_types[total_args];
-	char arg_modes[total_args];
+	std::vector<OID> arg_types(total_args);
+	std::vector<char> arg_modes(total_args);
```

Comment:
It will be better to use some string here too

Reply:
I think that vector is more suitable here because usage of string implies that you actually imply a string. This is an array of not related to each other chars